index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My-portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com" crossorigin="">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=poppins:wght@600&display=swap">
+
+</head>
+<body>
+    <header class="header">
+		<section class="nav">
+			<nav class="logo">
+			   <a href=""><h1>My Logo</h1></a>
+			</nav>
+			<nav class="burger">
+                <ul>
+                    <li>
+                        <label id="checkbox_toggle" class="burger">&#9776;</label>
+                    </li>
+                 </ul>
```

Comment:
- [ ] There is no use of this list element right now. Kindly, remove it since you are not gonna need it(YAGNI).